var/spack/repos/builtin/packages/pvm/package.py

```diff
@@ -17,19 +17,30 @@ class Pvm(MakefilePackage):
 
     version('3.4.6', sha256='482665e9bc975d826bcdacf1df1d42e43deda9585a2c430fd3b7b7ed08eada44')
 
+    depends_on('m4', type='build')
+    depends_on('libtirpc', type='link')
+
     parallel = False
 
     @property
     def pvm_arch(self):
         """Returns the appropriate PVM_ARCH."""
         process = subprocess.Popen(['lib/pvmgetarch'], stdout=subprocess.PIPE)
-        return process.communicate()[0].strip()
+        return process.communicate()[0].strip().decode()
 
     def edit(self, spec, prefix):
         # Before building PVM, you must set the environment
         # variable "PVM_ROOT" to the path where PVM resides
         env['PVM_ROOT'] = self.stage.source_path
 
+    def setup_build_environment(self, spack_env):
+        tirpc = self.spec['libtirpc'].prefix
+        spack_env.prepend_path(
+            'SPACK_INCLUDE_DIRS',
+            join_path(tirpc, 'include', 'tirpc'),
+        )
+        spack_env.set('SPACK_LDLIBS', '-ltirpc')
```

Comment:
```suggestion
    def setup_build_environment(self, env):
        tirpc = self.spec['libtirpc'].prefix
        env.prepend_path(
            'SPACK_INCLUDE_DIRS',
            tirpc.include.tirpc,
        )
        env.set('SPACK_LDLIBS', '-ltirpc')
```

Reply:
@adamjstewart - thanks for the comment! I will keep that in my for the future. Changes are pushed.